packages/@aws-cdk/aws-glue-alpha/test/integ.job-python-shell.js.snapshot/aws-glue-job-python-shell.template.json

```diff
@@ -108,113 +108,51 @@
      }
     },
     "DefaultArguments": {
-     "--job-language": "python",
-     "arg1": "value1",
-     "arg2": "value2"
+     "library-set": "analytics"
     },
-    "GlueVersion": "1.0",
+    "GlueVersion": "3.0",
     "MaxCapacity": 0.0625,
-    "Name": "ShellJob",
+    "MaxRetries": 0,
     "Role": {
      "Fn::GetAtt": [
-      "ShellJobServiceRoleCF97BC4B",
+      "IAMServiceRole61C662C4",
       "Arn"
      ]
-    },
-    "Tags": {
-     "key": "value"
     }
    }
   },
-  "ShellJob39ServiceRole2F6F3768": {
-   "Type": "AWS::IAM::Role",
+  "BasicShellJobC7D0761E": {
+   "Type": "AWS::Glue::Job",
    "Properties": {
-    "AssumeRolePolicyDocument": {
-     "Statement": [
-      {
-       "Action": "sts:AssumeRole",
-       "Effect": "Allow",
-       "Principal": {
-        "Service": "glue.amazonaws.com"
-       }
-      }
-     ],
-     "Version": "2012-10-17"
-    },
-    "ManagedPolicyArns": [
-     {
+    "Command": {
+     "Name": "pythonshell",
+     "PythonVersion": "3",
+     "ScriptLocation": {
       "Fn::Join": [
        "",
        [
-        "arn:",
+        "s3://",
         {
-         "Ref": "AWS::Partition"
+         "Fn::Sub": "cdk-hnb659fds-assets-${AWS::AccountId}-${AWS::Region}"
         },
-        ":iam::aws:policy/service-role/AWSGlueServiceRole"
+        "/432033e3218068a915d2532fa9be7858a12b228a2ae6e5c10faccd9097b1e855.py"
```

Comment:
this is strange? :D

Reply:
The files completely generated for the new Cfn resource. This will be massive colision